remediation/machine-deletion/configuration.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Configuration
+parent: Machine Deletion
+grand_parent: Remediation
+nav_order: 2
+---
+
+# Machine Deletion Remediation Configuration
+Machine Deletion Remediation (MDR) uses a Custom Resource (CR) created by [Node Health Check](https://github.com/medik8s/node-healthcheck-operator#readme) (NHC) based on a specific template.
+Since NHC comes with Self Node Remediation as the default remediation mechanism, it needs to be configured to use MDR; hence the following is required:
+1. Load the YAML manifest of the [MDR template](https://github.com/medik8s/machine-deletion#example-crs)
```

Comment:
I am not sure if load is the right word I would use here. Maybe `Add`

Reply:
I like `Add`. I will change it also in MDR Readme

Reply:
we do not load or add manifests, we create resources / CRs 🙂  I'd suggest something like:
`Create the MDR template CR using the provided [manifest](https://github.com/medik8s/machine-deletion#example-crs)`